go/teams/rpc_exim.go

```diff
@@ -45,6 +45,7 @@ func (t *Team) ExportToTeamPlusApplicationKeys(ctx context.Context, idTime keyba
 	writers = append(writers, members.Admins...)
 	writers = append(writers, members.Owners...)
 	onlyReaders = append(onlyReaders, members.Readers...)
+	onlyReaders = append(onlyReaders, members.Bots...)
```

Comment:
onlyReaders -> onlyBots
which would then be unused

Reply:
this would exclude `Bots` from being able to read kbfs (only one that uses this afaik) which they should be able to

Reply:
oh ok